etc/nyx.profile

```diff
@@ -0,0 +1,52 @@
+# Firejail profile for nyx
+# Description: Command-line status monitor for tor
+# This file is overwritten after every install/update
+# Persistent local customizations
+include nyx.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${PATH}/python2*
+noblacklist ${PATH}/python3*
+noblacklist /usr/lib/python2*
+noblacklist /usr/lib/python3*
+
+noblacklist ${HOME}/.nyx
+mkdir ${HOME}/.nyx
+whitelist ${HOME}/.nyx
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+caps.drop all
+ipc-namespace
+netfilter
+no3d
+nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+
+disable-mnt
+private-bin nyx,python*
+private-cache
+private-dev
+private-etc passwd,tor,fonts
+private-opt none
+private-srv none
```

Comment:
Not a review, just a comment -- I wonder about the usefulness of using `none` for private-* filters by default? I *think* it's something I've tended to avoid by default, but maybe we should use it more frequently???